plugins/plugin-chart-echarts/src/BoxPlot/controlPanel.ts

```diff
@@ -37,6 +35,7 @@ export default {
       controlSetRows: [
         ['metrics'],
         ['adhoc_filters'],
+        emitFilterControl,
```

Comment:
We should use `sections.emitFilterControl` for consistency namespace.

Reply:
I think this is ok - IMO `sections`  should be reserved for `ControlPanelSectionConfig` objects, not `ControlSetRow` objects.